### ISSUE TITLE
Multiple cucumber html reporter with CodeceptJS

### ISSUE THREAD

**Issue by xyngfei** [codeceptjs/CodeceptJS#2643]
### What are you trying to achieve?
Current HTML reporter does not support BDD format.

#### What do you get instead?
I want to use [multiple cucumber html reporter](https://www.npmjs.com/package/multiple-cucumber-html-reporter) with CodeceptJS.

I'm writing my tests in BDD which CodeceptJS supports but the report (Mochawesome) doesn't provide a detailed report and I don't prefer to use Allure because it needs to start the webserver to launch the report. 

I found this multiple cucumber html reporter meet my requirement and I wish to integrate this library into CodeceptJS reporter.

Could anyone please suggest how can I do this as I'm not able to see any JSON file being generated by CodeceptJS which is as per the requirement of this plugin?

**Comment by lobmailz** [comment]
Hi,

Were you able to integrate this reporter with codeceptjs?
Can you please suggest an approach?

**Comment by xyngfei** [comment]
> Hi,
> 
> Were you able to integrate this reporter with codeceptjs?
> Can you please suggest an approach?

No, end up I use [allure plugin](https://codecept.io/plugins/) to overcome this issue. 

FYI, you only can use it on devDependencies, because it may flag some vulnerabilities if you use nexusIQ to scan.
